docs/plaindate.md

```diff
@@ -71,13 +71,13 @@ If the value is any other object, it:
 - Must have either a number `month` property or a string `monthCode` property.
 - May have a `calendar` property. If omitted, the [ISO 8601 calendar](https://en.wikipedia.org/wiki/ISO_8601#Dates) will be used by default.
 
-If the value is not an object, it must be a string, which is expected to be in ISO 8601 format.
+If the value is not an object, it must be a string, which is expected to be in RFC 9557 format; roughly a superset of ISO 8601 format.
```

Comment:
This is a little ambiguous. One way to read it is that a full RFC 9557 string is required, but only the date portion is required. 

Maybe this sentence could be clearer, something like this:

> ...which is expected to be in RFC 9557 format, or a subset of that format that includes at least the date portion. Note that ISO 8601 and RFC 3339 formats are also subsets of RFC 9557, so will also work, except as noted below.

Reply:
Might also be nice to give an example here, e.g. 

> ...which is expected to be in RFC 9557 format, or a subset of that format that includes at least the date portion, for example `"2024-12-25"`. Note that ISO 8601 and RFC 3339 formats are also subsets of RFC 9557, so will also work, except as noted below.  Also note that Temporal accepts a few rare cases that RFC9557 does not, like the use of 6-digit years.  For more info, see {link here}.